javascript/javascript1/week2/Candy helper.js

```diff
@@ -0,0 +1,40 @@
+let boughtCandyPrices = [];
+let candyTablePrices = {
+    'sweet' : 0.5,
+    'chocolate' : 0.7,
+    'toffee' : 1.1,
+    'chewing-gum' : 0.03
+};
+
+function addCandy(candyType, weight) {
+    for (let key in candyTablePrices) {
+        if (candyType === key){
+            boughtCandyPrices.push(candyTablePrices[key] * weight);
```

Comment:
but what if i try to buy the wrong candy? there should be an else here :)

Reply:
Hello,
Thanks for the review. I'll make some corrections